test/test_helper.exs

```diff
@@ -3,7 +3,7 @@ File.rm_rf!(ArchEthic.Utils.mut_dir())
 ExUnit.start(
   exclude: [:infrastructure, :CI, :CD, :oracle_provider],
   timeout: :infinity,
-  max_failures: 1
+  max_failures: 5
```

Comment:
Why did you change the max failures to 5 ?

Reply:
This for testing personal modification.... I should have changed back to 1, but forgot. Will do that